src/popup/router/components/Modals/ConfirmTip.vue

```diff
@@ -3,12 +3,13 @@
     has-close-button
     @close="resolve"
   >
-    <h2 class="text-heading-2 text-center confirmation-title">
+    <!--     TODO - pawel - remove probably -->
+    <template slot="header">
       {{ $t('modals.confirm-tip.title') }}
-    </h2>
+    </template>
 
-    <p>{{ $t('modals.confirm-tip.msg') }}</p>
-    <p class="confirmation-question">
+    {{ $t('modals.confirm-tip.msg') }}
+    <p class="confirmation--question">
```

Comment:
I would be really happy to use BEM, but we are not using it currently. So I think one dash should be used.